drivers/net/ethernet/realtek/r8168/r8168_n.c

```diff
@@ -24002,6 +24004,11 @@ rtl8168_init_software_variable(struct net_device *dev)
 
         tp->NicCustLedValue = RTL_R16(tp, CustomLED);
 
+        if(tp->NicCustLedValue != RADXA_CUSTOM_LED) {
+                printk("status = 0x%x, RADXA_STATUS = 0x%x\r\n", tp->NicCustLedValue, RADXA_CUSTOM_LED);
+                RTL_W16(tp, CustomLED, RADXA_CUSTOM_LED);
+        }
+
```

Comment:
```suggestion
        RTL_W16(tp, CustomLED, RADXA_CUSTOM_LED);
        tp->NicCustLedValue = RTL_R16(tp, CustomLED);
```
不过最好是添加一个设备树属性来override这个值吧